DESCRIPTION

```diff
@@ -1,12 +1,13 @@
 Package: suntools
-Version: 1.1-8
-Date: 2023-04-15
+Version: 1.2.0
+Date: 2023-06-15
 Title: Tools for Handling Spatial Objects
 Authors@R: c(
 	person("Roger", "Bivand", role = c("cre", "aut"), email = "Roger.Bivand@nhh.no", comment=c(ORCID="0000-0003-2392-6140")),
```

Comment:
Roger doesn't want to be maintainer ("cre") anymore, so you can move that to my name. We also need to decide between aut or ctb role, here a reference on that: https://journal.r-project.org/archive/2012/RJ-2012-009/RJ-2012-009.pdf